From 5c2d219af445f1e8eed56556300475bfc2d633e9 Mon Sep 17 00:00:00 2001
From: Jeroen Hofstee <jhofstee@victronenergy.com>
Date: Wed, 10 Oct 2018 16:47:51 +0200
Subject: [PATCH 1/2] Revert "In non-DVCC systems, update the solarchargers at
 the same frequency."

This reverts commit edd71ff65d118f4a38300ae5d3ac8b1238af6e02.

Postponing this till the next release.
---
 delegates/dvcc.py | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/delegates/dvcc.py b/delegates/dvcc.py
index 9ef665d..76c4dca 100644
--- a/delegates/dvcc.py
+++ b/delegates/dvcc.py
@@ -640,9 +640,10 @@ class Dvcc(SystemCalcDelegate):
 	def _on_timer(self):
 		bol_support = self._settings['bol'] == 1
 
-		self._tickcount -= 1; self._tickcount %= ADJUST
-
 		if not bol_support:
+			# Update subsystems a bit less frequently. No reason than that is
+			# roughly how often we used to do it.
+			self._tickcount -= 1; self._tickcount %= (ADJUST*3)
 			if self._tickcount > 0: return True
 
 			voltage_written, current_written = self._legacy_update_solarchargers()
@@ -655,6 +656,7 @@ class Dvcc(SystemCalcDelegate):
 
 
 		# BOL/DVCC support below
+		self._tickcount -= 1; self._tickcount %= ADJUST
 
 		# Update subsystems
 		self._solarsystem.update_values()
-- 
2.7.4

